adaptor-rm-webapp/src/main/java/com/sample/rm/servlet/Application.java

```diff
@@ -88,7 +90,18 @@
         RESOURCE_CLASSES.add(ResourceShapeService.class);
 
         // Start of user code Custom Resource Classes
-            // End of user code
+
+        // trigger Jena init
+        ModelFactory.createDefaultModel();
+        // force plain XML writer
+        RDFWriterFImpl.alternative(null);
```

Comment:
Is this now out of date based on:
We will:
1. Depend on 'apache-jena-libs' instead of the individual artefacts.
2. Replace calls to 'model.getWriter("RDF/XML")' with the invocation of the RDFDataManager 'RDFDataMgr.write(outputStream, model, RDFFormat.RDFXML_PLAIN);'.

Reply:
replacing model.getwriter() in the jena-provider project seems to be the proper way to go. If we want to do that, @berezovskyi is arguing that we should do a proper refactoring of the provider classes in the jena-provider project. Do you have the time/effort to do it?
There are many Jax-RS providers that handle XML/RDF. but from my analysis of these classes, they all end up in the AbstractOslcRdfXmlProvider.java class.

The alternative above is the hack because it is based on private classes of Jena (within an adaptor) that gives you the same result, in the short term, without touching Jena-provider.